easybuild/easyconfigs/m/METIS/METIS-5.1.0-intel-2014b-32bitIDX.eb

```diff
@@ -0,0 +1,23 @@
+name = 'METIS'
+version = '5.1.0'
+versionsuffix = '-32bitIDX'
```

Comment:
can you clarify this versionsuffix?

The only 'special' thing I see in here is that the METIS shared libs get built?